Error message is not friendly

```
$ gjbt .
FAIL    github.com/hajimehoshi/ebiten   1.395s
warning: system calls not available, see https://github.com/gopherjs/gopherjs/blob/master/doc/syscalls.md
```

I understand syscall is not available, but would it be possible to have a better error message to know where syscall is used?

Hmm, that's interesting. If I deliberately introduce a failure into `myitcv.io/react` I do get some output:

```
$ gjbt myitcv.io/react
FAIL    myitcv.io/react 0.615s
--- FAIL: TestFail (0.01s)
        <anonymous>:38088: This should fail
FAIL
```

Do you have a commit pushed up somewhere that I can help debug this?

--- a/main.go
+++ b/main.go
@@ -98,7 +98,7 @@ func main() {
 			panic(err)
 		}
 
-		args := []string{"test", "--tags", *fTags, "-c", "-o", tf.Name(), "-v"}
+		args := []string{"test", "--tags", *fTags, "-c", "-o", tf.Name()}
 
 		args = append(args, pkg)
 
@@ -160,7 +160,7 @@ func main() {
 			fmt.Println(ec.Error)
 		}
 		fmt.Printf("%s\t%s\t%.3fs\n", status, bpkg.ImportPath, time.Since(start).Seconds())
-		
+
 		logs, err := p.ReadNewLogs("browser")
 		if err != nil {
 			panic(err)

--- a/main.go
+++ b/main.go
@@ -98,7 +98,7 @@ func main() {
 			panic(err)
 		}
 
-		args := []string{"test", "--tags", *fTags, "-c", "-o", tf.Name()}
+		args := []string{"test", "--tags", *fTags, "-c", "-o", tf.Name(), "-v"}
 
 		args = append(args, pkg)
 
@@ -156,8 +156,11 @@ func main() {
 			failed = true
 		}
 
+		if ec.Error != "" {
+			fmt.Println(ec.Error)
+		}
 		fmt.Printf("%s\t%s\t%.3fs\n", status, bpkg.ImportPath, time.Since(start).Seconds())
-
+		
 		logs, err := p.ReadNewLogs("browser")
 		if err != nil {
 			panic(err)

